### PR TITLE
fix/Add descriptive text to the General Settings page #3445

### PR DESCRIPTION
**Pull request by pinkelephant4**
Issue Number: #3445 
## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Application/ApplicationForm.tsx [hunk 1]
```diff
@@ -105,6 +105,7 @@ function ApplicationForm({ match }: Props) {
                 <Form>
                   <div className={`${CLASS_NAME}__header`}>
                     <h3>General Settings</h3>
+                    <h5>Enter a name and description for your app, and choose your app color.</h5>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Application/ApplicationForm.tsx:108]:
```suggestion
                    <h5>Enter a name and description for your app, and select a color.</h5>
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

**Comment by yuval-hazaz** [comment]
Thanks for your first contribution to Amplication

**Review by yuval-hazaz — APPROVED** [review]

**Comment by souravjain540** [comment]
your name and email @pinkelephant4 ?

**Comment by pinkelephant4** [comment]
> your name and email @pinkelephant4 ?

Ananya Maheshwari
maheshwari.ananya99@gmail.com

**Pull request merged**